task_12/src/css/style.css

```diff
@@ -0,0 +1,234 @@
+html, body, div, span, applet, object, iframe, h1, h2, h3, h4, h5, h6, p, blockquote, pre, a, abbr, acronym, address, big, cite, code, del, dfn, em, img, ins, kbd, q, s, samp, small, strike, strong, sub, sup, tt, var, b, u, i, center, dl, dt, dd, ol, ul, li, fieldset, form, label, legend, table, caption, tbody, tfoot, thead, tr, th, td, article, aside, canvas, details, embed, figure, figcaption, footer, header, hgroup, menu, nav, output, ruby, section, summary, time, mark, audio, video {
```

Comment:
why to `reset` files exists?

Reply:
It is convenient for me to divide into files so that it would be easier to change styles if necessary.
Therefore, reset is rendered in a separate file.
It seems to me that this does not affect it is in style file or imported there.

Thank you for noticing. Perhaps I missed something.